### PR TITLE
SONAR-8505 Implement smooth transition between pages

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

File: server/sonar-web/src/main/js/app/components/nav/global/GlobalNavMenu.js [hunk 1]
```diff
@@ -36,48 +37,53 @@ export default class GlobalNavMenu extends React.Component {
...
+        <li>
+          <Link to={{ pathname }} activeClassName="active">
+            {translate('projects.page')}
+          </Link>
         </li>
     );
   }
 
   renderIssuesLink () {
+    // TODO use <Link/>
```
Comment (SonarTech) [inline server/sonar-web/src/main/js/app/components/nav/global/GlobalNavMenu.js]: INFO Complete the task associated to this TODO comment. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 1 major
* INFO 1 info

Watch the comments in this conversation to review them.

**Comment by julienlancelot** [comment]
No sure it's related to this PR, but when updating the quality gate of a project, the green message is displayed twice : https://www.evernote.com/l/ACSBMIIRmrxNVqLznDTANO4F5PYyy0TjrOk

**Comment by julienlancelot** [comment]
On the account page, going on the notifications broke next navigation => The Notification link should be "disabled", waiting for this feature to be back

**Comment by stas-vilchik** [comment]
@julienlancelot 

- top messages are now disappear after some time, also it's possible to manually close them
- notifications page is disabled in master (I rebased this PR)

**Pull request merged**